### ISSUE TITLE
SIEdit crashes when opening JUKEBOX.SI

### ISSUE THREAD

**Issue by MatiLol251** [itsmattkc/SIEdit#12]
When i open JUKEBOX.SI, SIEdit does nothing at all, before it closes itself.
I load it using "Open" and "View SI File" and none of that works
The weirdest part is that if i open files such as "NOCD.SI" or "INTRO.SI" the app doesn't crash and can visualize the data correctly
Now i wonder why that doesn't happen with "JUKEBOX.SI"..

**Comment by itsmattkc** [comment]
I can't recreate this on my end - my JUKEBOX.SI opens fine.

Can you share what version/language of LEGO Island you're using? Or alternatively, send your JUKEBOX.SI file in some way?

**Comment by MatiLol251** [comment]
it's the spanish version

**Comment by MatiLol251** [comment]
maybe it's the fact that the files that contain voices are in higher quality than the english version. unfortunately, that means that the file is 91mb. No joke

**Comment by itsmattkc** [comment]
Confirmed the Spanish JUKEBOX.SI crashes libweaver. Will look into a fix for this.

**Comment by foxtacles** [comment]
I've investigated this issue a couple weeks back and we've found that the issue is in fact that the Spanish `JUKEBOX.SI` is corrupted. At file offset `0x2be0000`, where normally a split chunk is expected, one can find a block of 2048 zero bytes. This trips up SiEdit, as well as the original game itself:

https://github.com/isledecomp/SIEdit/assets/1135351/782b3649-cdc4-4fb0-973f-561ecded8ad2

It would be possible to make SiEdit more "lenient" to accomodate for this faulty file though if we want to work around it.

**Comment by MatiLol251** [comment]
Yes, I heard that tripping, but i thought it was part of the original release so I didn't think anything of it.
